pkg/service/deploy.go

```diff
@@ -95,22 +126,36 @@ func (s *DeployService) createTFMainFile(overrides *DeployOverrides, usecached b
 }
 
 func (s *DeployService) createDceInfra(ctx context.Context, overrides *DeployOverrides) (string, error) {
+	cfg := ctx.Value(constants.DeployConfig).(*configs.DeployConfig)
+
 	_, originDir := s.Util.ChToConfigDir()
 	defer s.Util.Chdir(originDir)
 
 	s.retrieveTFModules()
```

Comment:
we're missing an error handler here
(as a side note, we should get our updated linters running in this repo too, at some point)

Reply:
Technically so, but is an internal method and is called by ourselves and covered by unit tests.

Reply:
OTOH, that context is passed in by a method that is exposed externally. I'll add err handling there...